maintenance.md

```diff
@@ -804,8 +804,10 @@ Only the base image and module dependency should be updated.
 2. Check the upstream `Makefile` and `docker/Dockerfile`. If they have been updated significantly, update our `Dockerfile`.
    - https://github.com/gravitational/teleport/blob/vX.Y.Z/Makefile
    - https://github.com/gravitational/teleport/blob/vX.Y.Z/docker/Dockerfile
-3. Update `TELEPORT_VERSION` in `Dockerfile`.
-4. Update `BRANCH` and `TAG` files.
+3. Update `NODE_VERSION` in `Dockerfile`. Pleas reference bellow.
```

Comment:
It would be better to write the exact instruction.

```suggestion
3. Update `NODE_VERSION` in `Dockerfile`. Please use the version in the below file.
```

Reply:
Thank you for your comment.
I fixed it.